os_is_not_exist_matcher.go

```diff
@@ -0,0 +1,32 @@
+package gomegamatchers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/onsi/gomega/types"
+)
+
+func BeAnOsIsNotExistError() types.GomegaMatcher {
```

Comment:
Ah gotcha, that makes sense.

On Wednesday, June 15, 2016, Ben Moss <notifications@github.com> wrote:

> In os_is_not_exist_matcher.go
> <https://github.com/pivotal-cf-experimental/gomegamatchers/pull/3#discussion_r67187238>
> :
>
> > @@ -0,0 +1,32 @@
> > +package gomegamatchers
> > +
> > +import (
> > +	"fmt"
> > +	"os"
> > +
> > +	"github.com/onsi/gomega/types"
> > +)
> > +
> > +func BeAnOsIsNotExistError() types.GomegaMatcher {
>
> There's still a constructor function for BeFalse:
> https://github.com/onsi/gomega/blob/3e93f011213c200aadafcb19ed56eb718381cfbd/matchers.go#L49-L51
>
> Without the constructor you would have to express assertions as
> Expect(true).To(BeFalseMatcher{}).
>
> It looks like the Gomega codebase puts all the constructors in a separate
> file
> <https://github.com/onsi/gomega/blob/3e93f011213c200aadafcb19ed56eb718381cfbd/matchers.go>,
> whereas this codebase has them in the same file as the matcher structs.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/pivotal-cf-experimental/gomegamatchers/pull/3/files/2fae31d39f8522eba9d417f3f297bec8d9c4ca13#r67187238>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC9t3hQSfqTgHEyP3dT6Y738J94oVfNxks5qMB1LgaJpZM4I1aZT>
> .
>